packages/core/src/repositories/commitsRepository/utils/getHeadCommit.ts

```diff
@@ -27,5 +27,5 @@ export async function getHeadCommitForProject(
     .orderBy(desc(commitsScope.mergedAt))
     .limit(1)
 
-  return Result.ok(result[0])
+  return result[0] as Commit | undefined
```

Comment:
this can clearly be undefined yet types always asserted commit existed, for some reason

Reply:
Just don't do the `as` no? It will give drizzle schema or `undefined`

Reply:
for some reason it doesn't, not sure what is happening here